source/operations/install-deploy-manage/upgrade-minio-operator.rst

```diff
@@ -36,15 +39,147 @@ The following table lists the upgrade paths from previous versions of the MinIO
 
 .. _minio-k8s-upgrade-minio-operator-procedure:
 
-Upgrade MinIO Operator 4.2.3 and Later to |operator-version-stable|
+Upgrade MinIO Operator 4.5.8 and Later to |operator-version-stable|
 -------------------------------------------------------------------
 
 .. admonition:: Prerequisites
    :class: note
 
    This procedure requires the following:
 
-   - You have an existing MinIO Operator deployment running 4.2.3 or later
+   - You have an existing MinIO Operator deployment running 4.5.8 or later
+   - Your Kubernetes cluster runs 1.19.0 or later
+   - Your local host has ``kubectl`` installed and configured with access to the Kubernetes cluster
+
+This procedure upgrades the MinIO Operator from any 4.5.8 or later release to |operator-version-stable|.
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+The following changes apply for Operator v5.0.0 or later:
+
+- The Operator's Graphical User Interface now comes bundled with the same container as the Operator.
+- The ``.spec.s3`` field is replaced by the ``.spec.features`` field.
+- The ``.spec.credsSecret`` field is replaced by the ``.spec.configuration`` field.
+
+  This secret should hold all the environment variables for the MinIO deployment that contain sensitive information and should not show in ``.spec.env``.
+- Both the **Log Search API** (``.spec.log``) and **Prometheus** (``.spec.prometheus``) deployments have been removed.
+  However, existing deployments are left running as standalone deployments / statefulsets with no connection to the Tenant CR.
+  If the Tenant CR is deleted, this does not cascade to the log or Prometheus deployments.
+
+  .. important::
+
+     MinIO recommends that you create a yaml file to manage these deployments going forward.
+
+.. container:: procedure
```

Comment:
Are we doing CSS here on this?

Maybe break this up into

H2. Tenant Custom Resource Definition Changes

H2. Log Search and Prometheus Removal

H2. Upgrade Procedure

So people can handle each thing in order.

Reply:
Not sure. I was just following what you had done last time you updated this page, which added this `.. procedure::` for each one. I can drop it.

Reply:
I can't even begin to remember why I did that. It must be an scss thing, but I think at this point clever things can go away for things that work.

Reply:
Removed the .. procedure throughout and added headings.